Client/mods/deathmatch/logic/lua/CLuaMain.cpp

```diff
@@ -227,6 +227,7 @@ bool CLuaMain::LoadScriptFromBuffer(const char* cpInBuffer, unsigned int uiInSiz
     // If compiled script, make sure correct chunkname is embedded
     CLuaShared::EmbedChunkName(strNiceFilename, &cpBuffer, &uiSize);
 
+    bool ret{};
```

Comment:
Do you need this value?

Reply:
It's set below on line 284 

https://github.com/multitheftauto/mtasa-blue/pull/2713/files#diff-5646d00f776d78bd8a35295ac5f410aa81465a191106fc8fa49a5d6f90952b3cR284

Reply:
Yeah, but just `return true` and `return false` are better